ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile

```diff
@@ -159,6 +160,24 @@ RUN cd /opt && \
     ./download-bigdl.sh && \
     rm bigdl*.zip
 
+# sbt
+RUN apt-get update && \
```

Comment:
If we just need TPC-H jar, please move docker build stage to bigdl build stage. Such that final stage and image will be much smaller.